swapi/src/scripts/dataModel.js

```diff
@@ -0,0 +1,39 @@
+class Model {
+    constructor() {
+		this.resultVolume = 5;
+    }
+
+    async getApiData(addUrl) {
+    	let apiData = await fetch(`https://swapi.dev/api/${addUrl}`)
+			.then((response) => 
+				response.json()
+			)
+			.catch(()=> {
+				alert("Error. Try refresh page.")
+			})
+    	return apiData
```

Comment:
why do you create a variable if you supposed to return result in then ?

Reply:
I fixed this.
```suggestion
 async getApiData(addUrl) {
    return await fetch(`https://swapi.dev/api/${addUrl}`)
      .then((response) => response.json())
      .catch((error) => {
        console.error(error);
      });
  }
```